changelog.d/20230822_155752_ada.rst

```diff
@@ -0,0 +1,4 @@
+Added
+~~~~~
+
+- Add support for pausing and resuming timers. (:pr:`NUMBER`)
```

Comment:
I like for the SDK changelogs to call out the method names when we add things. This is both for users and for us -- if we want to find the version in which `pause_job` was added, it's nice to be able to search the changelog for `pause_job`.

Exact phrasing is up to you, but I'd do it something like
> - Add ``TimerClient.pause_job`` and ``TimerClient.resume_job`` for pausing and resuming timers.